In [225]:
import pandas as pd
import matplotlib as plt
import seaborn as sns
import numpy as np

from scipy.io import loadmat

### loading Data
---

#### Loading user 01 data

In [226]:
#loading data for the user 01, first day frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_FreqD_FDay.mat")
# print(data.keys())
U01_Freq_FD= data['Acc_FD_Feat_Vec']
# print(Acc_FD_Feat_Vec)

df = pd.DataFrame(U01_Freq_FD)

df.head()


0         1         2         3         4         5         6   \
0  0.011701  0.695183  0.056053  0.852710  0.000361  0.054925  0.003017   
1  0.007605  0.365587  0.056786  0.890850  0.000211  0.056385  0.003179   
2  0.006774  0.317463  0.056913  0.896575  0.000597  0.056619  0.003206   
3  0.007355  0.257231  0.058783  0.930654  0.000594  0.058435  0.003415   
4  0.007774  0.333129  0.057743  0.908621  0.000932  0.057330  0.003287   

         7         8         9   ...        33        34        35        36  \
0  0.002456  0.014748  0.852350  ...  0.314792  0.000808  0.054925  0.000422   
1  0.001337  0.010256  0.890639  ...  0.358384  0.000119  0.056385  0.000522   
2  0.000905  0.009617  0.895978  ...  0.355267  0.000104  0.056619  0.000516   
3  0.000888  0.010486  0.930060  ...  0.346360  0.000208  0.058435  0.000507   
4  0.001382  0.010582  0.907689  ...  0.348034  0.000132  0.057330  0.000492   

         37        38        39        40        41        42  
0  0.001148  0.006112  0.313985  0.001215  0.000875  0.001148  
1  0.000180  0.004870  0.358265  0.000397  0.000131  0.000180  
2  0.000172  0.004834  0.355163  0.000524  0.000116  0.000172  
3  0.000312  0.006594  0.346152  0.000532  0.000227  0.000312  
4  0.000198  0.004975  0.347903  0.000369  0.000144  0.000198  

[5 rows x 43 columns]

In [227]:
df.shape

(36, 43)

In [228]:
#loading data for the user 01, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_FreqD_MDay.mat")
print(data.keys())
U01_Freq_MD= data['Acc_FD_Feat_Vec']
df= pd.DataFrame(U01_Freq_MD)
df.head()

df.describe()


dict_keys(['__header__', '__version__', '__globals__', 'Acc_FD_Feat_Vec'])


0          1          2          3          4          5   \
count  36.000000  36.000000  36.000000  36.000000  36.000000  36.000000   
mean    0.007035   0.286527   0.058898   0.927687   0.000384   0.058583   
std     0.000950   0.088815   0.001624   0.028479   0.000355   0.001646   
min     0.005892  -0.012995   0.056735   0.893680   0.000013   0.056478   
25%     0.006429   0.262312   0.057904   0.909737   0.000163   0.057536   
50%     0.007022   0.302280   0.058611   0.922414   0.000307   0.058198   
75%     0.007440   0.318297   0.059198   0.933194   0.000510   0.058873   
max     0.011686   0.587452   0.064587   1.030423   0.001897   0.064401   

              6          7          8          9   ...         33         34  \
count  36.000000  36.000000  36.000000  36.000000  ...  36.000000  36.000000   
mean    0.003435   0.000598   0.010463   0.927303  ...   0.343634   0.000198   
std     0.000197   0.000491   0.000880   0.028514  ...   0.019702   0.000141   
min     0.003190   0.000045   0.009295   0.893194  ...   0.288194   0.000030   
25%     0.003310   0.000262   0.009950   0.909529  ...   0.336185   0.000091   
50%     0.003387   0.000468   0.010385   0.921927  ...   0.347078   0.000144   
75%     0.003466   0.000770   0.010725   0.932826  ...   0.355465   0.000312   
max     0.004147   0.002679   0.014649   1.029970  ...   0.377845   0.000500   

              35         36         37         38         39         40  \
count  36.000000  36.000000  36.000000  36.000000  36.000000  36.000000   
mean    0.058583   0.000486   0.000304   0.005164   0.343436   0.000540   
std     0.001646   0.000047   0.000195   0.000585   0.019729   0.000253   
min     0.056478   0.000357   0.000051   0.004433   0.287694   0.000085   
25%     0.057536   0.000463   0.000148   0.004759   0.335928   0.000309   
50%     0.058198   0.000493   0.000237   0.004932   0.346918   0.000562   
75%     0.058873   0.000516   0.000461   0.005472   0.355192   0.000705   
max     0.064401   0.000571   0.000709   0.006693   0.377535   0.001030   

              41         42  
count  36.000000  36.000000  
mean    0.000222   0.000304  
std     0.000152   0.000195  
min     0.000034   0.000051  
25%     0.000101   0.000148  
50%     0.000171   0.000237  
75%     0.000339   0.000461  
max     0.000536   0.000709  

[8 rows x 43 columns]

shape of the frequency data are
(36,43)

In [229]:
#loading data for the user 01, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_TimeD_FDay.mat")
print(data.keys())
U01_Time_FD= data['Acc_TD_Feat_Vec']
df= pd.DataFrame(U01_Time_FD)
df.head()

dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


0             1         2         3         4         5         6   \
0  1.259237  9.776725e-11  0.269896  0.852272  0.072844  0.904810  0.198181   
1  1.211754  5.020433e-01  0.175407  0.896821  0.030768  0.919309  0.151158   
2  1.161662  5.706772e-01  0.137488  0.917659  0.018903  0.940830  0.115722   
3  1.295694  5.489352e-01  0.165845  0.911959  0.027504  0.919734  0.142006   
4  1.174379  6.007878e-01  0.154973  0.913624  0.024017  0.922476  0.134560   

         7         8         9   ...   78   79   80   81   82    83     84  \
0  1.259237  0.893851 -0.650420  ...  0.0  0.0  0.0  0.0  0.0  45.0  163.0   
1  0.709710  0.913758 -0.556562  ...  0.0  0.0  0.0  0.0  0.0  21.0  170.0   
2  0.590985  0.927867 -0.405163  ...  0.0  0.0  0.0  0.0  0.0  25.0  161.0   
3  0.746758  0.926866 -0.274121  ...  0.0  0.0  0.0  0.0  0.0  20.0  169.0   
4  0.573591  0.926631 -0.491495  ...  0.0  0.0  0.0  0.0  0.0  22.0  173.0   

      85   86   87  
0   92.0  0.0  0.0  
1  109.0  0.0  0.0  
2  114.0  0.0  0.0  
3  111.0  0.0  0.0  
4  105.0  0.0  0.0  

[5 rows x 88 columns]

In [230]:
df.shape

(36, 88)

shape of the time domain features are (36,88)

In [231]:
#loading data for the user 01, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_TimeD_MDay.mat")
print(data.keys())
U01_Time_MD= data['Acc_TD_Feat_Vec']
df= pd.DataFrame(U01_Time_FD)
df.head()

dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


0             1         2         3         4         5         6   \
0  1.259237  9.776725e-11  0.269896  0.852272  0.072844  0.904810  0.198181   
1  1.211754  5.020433e-01  0.175407  0.896821  0.030768  0.919309  0.151158   
2  1.161662  5.706772e-01  0.137488  0.917659  0.018903  0.940830  0.115722   
3  1.295694  5.489352e-01  0.165845  0.911959  0.027504  0.919734  0.142006   
4  1.174379  6.007878e-01  0.154973  0.913624  0.024017  0.922476  0.134560   

         7         8         9   ...   78   79   80   81   82    83     84  \
0  1.259237  0.893851 -0.650420  ...  0.0  0.0  0.0  0.0  0.0  45.0  163.0   
1  0.709710  0.913758 -0.556562  ...  0.0  0.0  0.0  0.0  0.0  21.0  170.0   
2  0.590985  0.927867 -0.405163  ...  0.0  0.0  0.0  0.0  0.0  25.0  161.0   
3  0.746758  0.926866 -0.274121  ...  0.0  0.0  0.0  0.0  0.0  20.0  169.0   
4  0.573591  0.926631 -0.491495  ...  0.0  0.0  0.0  0.0  0.0  22.0  173.0   

      85   86   87  
0   92.0  0.0  0.0  
1  109.0  0.0  0.0  
2  114.0  0.0  0.0  
3  111.0  0.0  0.0  
4  105.0  0.0  0.0  

[5 rows x 88 columns]

loading combined datasets

In [232]:
# combined dataset for time and frequency for user 01 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_TimeD_FreqD_FDay.mat")
print(data.keys())
U01_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']
df= pd.DataFrame(U01_TimeFq_FD)
df.head()


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


0             1         2         3         4         5         6    \
0  1.259237  9.776725e-11  0.269896  0.852272  0.072844  0.904810  0.198181   
1  1.211754  5.020433e-01  0.175407  0.896821  0.030768  0.919309  0.151158   
2  1.161662  5.706772e-01  0.137488  0.917659  0.018903  0.940830  0.115722   
3  1.295694  5.489352e-01  0.165845  0.911959  0.027504  0.919734  0.142006   
4  1.174379  6.007878e-01  0.154973  0.913624  0.024017  0.922476  0.134560   

        7         8         9    ...  121  122  123  124  125   126    127  \
0  1.259237  0.893851 -0.650420  ...  0.0  0.0  0.0  0.0  0.0  45.0  163.0   
1  0.709710  0.913758 -0.556562  ...  0.0  0.0  0.0  0.0  0.0  21.0  170.0   
2  0.590985  0.927867 -0.405163  ...  0.0  0.0  0.0  0.0  0.0  25.0  161.0   
3  0.746758  0.926866 -0.274121  ...  0.0  0.0  0.0  0.0  0.0  20.0  169.0   
4  0.573591  0.926631 -0.491495  ...  0.0  0.0  0.0  0.0  0.0  22.0  173.0   

     128  129  130  
0   92.0  0.0  0.0  
1  109.0  0.0  0.0  
2  114.0  0.0  0.0  
3  111.0  0.0  0.0  
4  105.0  0.0  0.0  

[5 rows x 131 columns]

In [233]:
print(U01_TimeFq_FD.shape)

(36, 131)


In [234]:
# combined dataset for time and frequency for user 01 on multple days
data = loadmat("../data/CW-Data/CW-Data/U01_Acc_TimeD_FreqD_MDay.mat")
print(data.keys())
U01_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']
df= pd.DataFrame(U01_TimeFq_FD)
df.head()

dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


0             1         2         3         4         5         6    \
0  1.259237  9.776725e-11  0.269896  0.852272  0.072844  0.904810  0.198181   
1  1.211754  5.020433e-01  0.175407  0.896821  0.030768  0.919309  0.151158   
2  1.161662  5.706772e-01  0.137488  0.917659  0.018903  0.940830  0.115722   
3  1.295694  5.489352e-01  0.165845  0.911959  0.027504  0.919734  0.142006   
4  1.174379  6.007878e-01  0.154973  0.913624  0.024017  0.922476  0.134560   

        7         8         9    ...  121  122  123  124  125   126    127  \
0  1.259237  0.893851 -0.650420  ...  0.0  0.0  0.0  0.0  0.0  45.0  163.0   
1  0.709710  0.913758 -0.556562  ...  0.0  0.0  0.0  0.0  0.0  21.0  170.0   
2  0.590985  0.927867 -0.405163  ...  0.0  0.0  0.0  0.0  0.0  25.0  161.0   
3  0.746758  0.926866 -0.274121  ...  0.0  0.0  0.0  0.0  0.0  20.0  169.0   
4  0.573591  0.926631 -0.491495  ...  0.0  0.0  0.0  0.0  0.0  22.0  173.0   

     128  129  130  
0   92.0  0.0  0.0  
1  109.0  0.0  0.0  
2  114.0  0.0  0.0  
3  111.0  0.0  0.0  
4  105.0  0.0  0.0  

[5 rows x 131 columns]

shape of the combined datasets are (36, 131) <br>
131 features are made from 43 frequency features and 88 time domain features. 

#### loading user 02 data 

In [235]:
# loading frequency data for user 02 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_FreqD_FDay.mat")
# print(data.keys())
U02_Freq_FD= data['Acc_FD_Feat_Vec']
# print(Acc_FD_Feat_Vec)

df = pd.DataFrame(U01_Freq_FD)

df.head()

0         1         2         3         4         5         6   \
0  0.011701  0.695183  0.056053  0.852710  0.000361  0.054925  0.003017   
1  0.007605  0.365587  0.056786  0.890850  0.000211  0.056385  0.003179   
2  0.006774  0.317463  0.056913  0.896575  0.000597  0.056619  0.003206   
3  0.007355  0.257231  0.058783  0.930654  0.000594  0.058435  0.003415   
4  0.007774  0.333129  0.057743  0.908621  0.000932  0.057330  0.003287   

         7         8         9   ...        33        34        35        36  \
0  0.002456  0.014748  0.852350  ...  0.314792  0.000808  0.054925  0.000422   
1  0.001337  0.010256  0.890639  ...  0.358384  0.000119  0.056385  0.000522   
2  0.000905  0.009617  0.895978  ...  0.355267  0.000104  0.056619  0.000516   
3  0.000888  0.010486  0.930060  ...  0.346360  0.000208  0.058435  0.000507   
4  0.001382  0.010582  0.907689  ...  0.348034  0.000132  0.057330  0.000492   

         37        38        39        40        41        42  
0  0.001148  0.006112  0.313985  0.001215  0.000875  0.001148  
1  0.000180  0.004870  0.358265  0.000397  0.000131  0.000180  
2  0.000172  0.004834  0.355163  0.000524  0.000116  0.000172  
3  0.000312  0.006594  0.346152  0.000532  0.000227  0.000312  
4  0.000198  0.004975  0.347903  0.000369  0.000144  0.000198  

[5 rows x 43 columns]

In [236]:
#loading data for the user 02, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_FreqD_MDay.mat")
print(data.keys())
U02_Freq_MD= data['Acc_FD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_FD_Feat_Vec'])


In [237]:
#loading data for the user 02, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_TimeD_FDay.mat")
print(data.keys())
U02_Time_FD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


In [238]:
#loading data for the user 02, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_TimeD_MDay.mat")
print(data.keys())
U02_Time_MD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


combined datasets

In [239]:
# combined dataset for time and frequency for user 02 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_TimeD_FreqD_FDay.mat")
print(data.keys())
U02_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 02 on multple days
data = loadmat("../data/CW-Data/CW-Data/U02_Acc_TimeD_FreqD_MDay.mat")
print(data.keys())
U02_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])
dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


#### loading user 03 data

In [240]:
# loading frequency data for user 03 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_FreqD_FDay.mat")
U03_Freq_FD= data['Acc_FD_Feat_Vec']
# print(Acc_FD_Feat_Vec)

In [241]:
#loading data for the user 03, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_FreqD_MDay.mat")
print(data.keys())
U03_Freq_MD= data['Acc_FD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_FD_Feat_Vec'])


In [242]:
#loading data for the user 03, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_TimeD_FDay.mat")
print(data.keys())
U03_Time_FD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


In [243]:
#loading data for the user 03, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_TimeD_MDay.mat")
print(data.keys())
U03_Time_MD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


combined datasets

In [244]:
# combined dataset for time and frequency for user 03 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_TimeD_FreqD_FDay.mat")
print(data.keys())
U03_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 02 on multple days
data = loadmat("../data/CW-Data/CW-Data/U03_Acc_TimeD_FreqD_MDay.mat")
print(data.keys())
U03_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])
dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


#### loading User 04 data

In [245]:
# loading frequency data for user 04 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_FreqD_FDay.mat")
# print(data.keys())
U04_Freq_FD= data['Acc_FD_Feat_Vec']


In [246]:
#loading data for the user 04, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_FreqD_MDay.mat")
print(data.keys())
U04_Freq_MD= data['Acc_FD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_FD_Feat_Vec'])


In [247]:
#loading data for the user 04, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_TimeD_FDay.mat")
print(data.keys())
U04_Time_FD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


In [248]:
#loading data for the user 04, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_TimeD_MDay.mat")
print(data.keys())
U04_Time_MD= data['Acc_TD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


combined datasets

In [249]:
# combined dataset for time and frequency for user 03 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_TimeD_FreqD_FDay.mat")
print(data.keys())
U04_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 03 on multple days
data = loadmat("../data/CW-Data/CW-Data/U04_Acc_TimeD_FreqD_MDay.mat")
print(data.keys())
U04_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])
dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


#### loading user 05 data

In [250]:
# loading frequency data for user 05 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_FreqD_FDay.mat")
U05_Freq_FD= data['Acc_FD_Feat_Vec']


In [251]:
#loading data for the user 05, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_FreqD_MDay.mat")
print(data.keys())
U05_Freq_MD= data['Acc_FD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_FD_Feat_Vec'])


In [252]:
#loading data for the user 05, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_TimeD_FDay.mat")
print(data.keys())
U05_Time_FD= data['Acc_TD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


In [253]:
#loading data for the user 05, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_TimeD_MDay.mat")
print(data.keys())
U05_Time_MD= data['Acc_TD_Feat_Vec']

dict_keys(['__header__', '__version__', '__globals__', 'Acc_TD_Feat_Vec'])


combined datasets

In [254]:
# combined dataset for time and frequency for user 05 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_TimeD_FreqD_FDay.mat")
print(data.keys())
U05_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 05 on multple days
data = loadmat("../data/CW-Data/CW-Data/U05_Acc_TimeD_FreqD_MDay.mat")
print(data.keys())
U05_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']


dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])
dict_keys(['__header__', '__version__', '__globals__', 'Acc_TDFD_Feat_Vec'])


#### loading user 06 data

In [255]:
# loading frequency data for user 06 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_FreqD_FDay.mat")
U06_Freq_FD= data['Acc_FD_Feat_Vec']

#loading data for the user 06, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_FreqD_MDay.mat")
U06_Freq_MD= data['Acc_FD_Feat_Vec']

#loading data for the user 06, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_TimeD_FDay.mat")
U06_Time_FD= data['Acc_TD_Feat_Vec']

#loading data for the user 06, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_TimeD_MDay.mat")
U06_Time_MD= data['Acc_TD_Feat_Vec']

# combined datasets

# combined dataset for time and frequency for user 06 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_TimeD_FreqD_FDay.mat")
U06_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 06 on multple days
data = loadmat("../data/CW-Data/CW-Data/U06_Acc_TimeD_FreqD_MDay.mat")
U06_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']


#### loading user 07 data

In [256]:
# loading frequency data for user 07 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_FreqD_FDay.mat")
U07_Freq_FD= data['Acc_FD_Feat_Vec']

#loading data for the user 07, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_FreqD_MDay.mat")
U07_Freq_MD= data['Acc_FD_Feat_Vec']

#loading data for the user 07, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_TimeD_FDay.mat")
U07_Time_FD= data['Acc_TD_Feat_Vec']

#loading data for the user 07, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_TimeD_MDay.mat")
U07_Time_MD= data['Acc_TD_Feat_Vec']

# combined datasets

# combined dataset for time and frequency for user 07 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_TimeD_FreqD_FDay.mat")
U07_TimeFq_FD = data ['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 07 on multple days
data = loadmat("../data/CW-Data/CW-Data/U07_Acc_TimeD_FreqD_MDay.mat")
U07_TimeFq_MD = data ['Acc_TDFD_Feat_Vec']

#### loading user 08 data

In [257]:
# loading frequency data for user 08 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_FreqD_FDay.mat")
U08_Freq_FD = data['Acc_FD_Feat_Vec']

# loading data for the user 08, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_FreqD_MDay.mat")
U08_Freq_MD = data['Acc_FD_Feat_Vec']

# loading data for the user 08, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_TimeD_FDay.mat")
U08_Time_FD = data['Acc_TD_Feat_Vec']

# loading data for the user 08, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_TimeD_MDay.mat")
U08_Time_MD = data['Acc_TD_Feat_Vec']

# combined datasets

# combined dataset for time and frequency for user 08 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_TimeD_FreqD_FDay.mat")
U08_TimeFq_FD = data['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 08 on multiple days
data = loadmat("../data/CW-Data/CW-Data/U08_Acc_TimeD_FreqD_MDay.mat")
U08_TimeFq_MD = data['Acc_TDFD_Feat_Vec']


#### loading user 09 data

In [258]:
# loading frequency data for user 09 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_FreqD_FDay.mat")
U09_Freq_FD = data['Acc_FD_Feat_Vec']

# loading data for the user 09, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_FreqD_MDay.mat")
U09_Freq_MD = data['Acc_FD_Feat_Vec']

# loading data for the user 09, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_TimeD_FDay.mat")
U09_Time_FD = data['Acc_TD_Feat_Vec']

# loading data for the user 09, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_TimeD_MDay.mat")
U09_Time_MD = data['Acc_TD_Feat_Vec']

# combined datasets

# combined dataset for time and frequency for user 09 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_TimeD_FreqD_FDay.mat")
U09_TimeFq_FD = data['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 09 on multiple days
data = loadmat("../data/CW-Data/CW-Data/U09_Acc_TimeD_FreqD_MDay.mat")
U09_TimeFq_MD = data['Acc_TDFD_Feat_Vec']

#### loading user 10 data

In [ ]:
# loading frequency data for user 10 on the same day (first day)
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_FreqD_FDay.mat")
U10_Freq_FD = data['Acc_FD_Feat_Vec']

# loading data for the user 10, multiple days frequency data. 
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_FreqD_MDay.mat")
U10_Freq_MD = data['Acc_FD_Feat_Vec']

# loading data for the user 10, first day time data. 
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_TimeD_FDay.mat")
U10_Time_FD = data['Acc_TD_Feat_Vec']

# loading data for the user 10, multiple days time data. 
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_TimeD_MDay.mat")
U10_Time_MD = data['Acc_TD_Feat_Vec']

# combined datasets

# combined dataset for time and frequency for user 10 on the first day (same day)
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_TimeD_FreqD_FDay.mat")
U10_TimeFq_FD = data['Acc_TDFD_Feat_Vec']

# combined dataset for time and frequency for user 10 on multiple days
data = loadmat("../data/CW-Data/CW-Data/U10_Acc_TimeD_FreqD_MDay.mat")
U10_TimeFq_MD = data['Acc_TDFD_Feat_Vec']


### EDA
---

so there are 60 datasets in total now.  each dataset has been loaded into an array. 
for extra clarity the file names assigned to the variables differ a bit in variable assignment here.

same day (first day) frequency domain acceleration data - U01_Freq_FD <br>
multiple days(cross days) frequency domain acceleration data - U01_Freq_MD <br>
same day (first day) time domain  acceleration data - U01_Time_FD <br>
multiple days (cross days) time domain acceleration data - U01_Time_MD

same day (first day) frquency and time domain acceleration data - U01_TimeFq_FD <br>
same day (first day) frquency and time domain acceleration data - U01_TimeFq_MD